OpenRA.Mods.Common/Widgets/WorldCommandWidget.cs

```diff
@@ -191,15 +191,28 @@ bool CycleBases()
 				.Select(b => b.Actor)
 				.ToList();
 
-			if (!bases.Any())
-				return true;
+            var noBase = false;
+
+            // If no base found the nondestroyed buildings are selected.
```

Comment:
"nondestroyed" -> "existing"
"If no base found" -> "If no base is found,"
aaand probably "the" -> "all" :D